data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/validator/PostBlock.java

```diff
@@ -64,7 +64,7 @@ public PostBlock(
           @OpenApiRequestBody(content = {@OpenApiContent(from = SignedBeaconBlock.class)}),
       description =
           "Submit a signed beacon block to the beacon node to be imported."
-              + " Validation is expected to be performed by the beacon node.",
+              + " Validation must be performed by the beacon node.",
```

Comment:
I'd go with 'will'

Reply:
"Will" is on out banned words list. But I've reworded it slightly to be less strongly worded.